### PR TITLE
Fixed tag model/controller annotations and added another method to re…

### PR DESCRIPTION
**Pull request by env7295**
…pository as well as some java docs and todo comments. Thank you for the help on some of these fixes Arjun.

### CODE REVIEW

File: src/main/java/com/revature/byteshare/Tags/TagController.java [hunk 1]
```diff
@@ -10,30 +10,49 @@
...
+    /*
+    @GetMapping("/{tag_name}") // possibly use query param? not entirely sure when to use one over the other tbh
+    public List<Recipe> getAllRecipesByTagName(@PathVariable String tag_name){
+        return
     }
-    @GetMapping("/{recipeId")
+     */
+
+    // TODO : FOR REVIEWER: do we actually just want this to return a list
+    //  of Tag_names or do we want it returning the whole tag object?
```
Comment (aramsing) [inline src/main/java/com/revature/byteshare/Tags/TagController.java:32]: Just the tag name will do

**Review by aramsing — COMMENTED** [review]
You can uncomment the method and make sure it works before we merge

File: src/main/java/com/revature/byteshare/Tags/TagController.java (hunk 1 above)
Comment (env7295) [reply src/main/java/com/revature/byteshare/Tags/TagController.java:32]: Sounds good, I changed that methods to only return the tag names.

**Review by aramsing — APPROVED** [review]
Looks good

**Pull request merged**